### PR TITLE
fix: Fix ItemGuessFeature tooltip line being fully underlined incorrectly [skip ci]

### PR DESCRIPTION
**Pull request by kristofbolyai**
# Old
image

# Fixed
image

### CODE REVIEW

File: common/src/main/java/com/wynntils/features/tooltips/ItemGuessFeature.java [hunk 1]
```diff
@@ -91,12 +97,14 @@ private List<Component> getTooltipAddon(GearBoxItem gearBoxItem) {
 
             guesses.append(StyledText.fromString("§7: ").getComponent());
 
-            Optional<MutableComponent> itemsComponent = itemsForLevel.stream()
-                    .reduce((i, j) -> i.append(Component.literal(", ").withStyle(ChatFormatting.GRAY))
-                            .append(j));
+            MutableComponent itemsComponent = Component.empty();
+            itemsComponent.append(itemsForLevel.get(0));
```
Comment (ShadowCat117) [inline common/src/main/java/com/wynntils/features/tooltips/ItemGuessFeature.java:102]: Won't this fail if `itemsForLevel` is empty? Maybe in the above check it should just append something like "Unknown item" or just skip to the next level

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/features/tooltips/ItemGuessFeature.java:102]: It should never be empty, but I forgot to add the continue after the logging above.

**Comment by magicus** [comment]
What does the underlining signify?

**Comment by kristofbolyai** [comment]
> What does the underlining signify?

Whether the item is favorited.

**Comment by magicus** [comment]
Ah, I see. I have not used favourites that much. Maybe we should insert a yellow ☆ in front instead? I think that would be more visible and intuitive?

**Comment by kristofbolyai** [comment]
> Ah, I see. I have not used favourites that much. Maybe we should insert a yellow ☆ in front instead? I think that would be more visible and intuitive?

We already have a star in front of the item itself, so underlining should be okay.

**Comment by magicus** [comment]
Okay. :-)

**Pull request merged**